Examples/Algorithms/Geant4/src/Geant4Simulation.cpp

```diff
@@ -84,6 +87,13 @@ ActsExamples::Geant4Simulation::Geant4Simulation(
   G4EventManager::GetEventManager()->GetStackManager()->SetVerboseLevel(
       geantVerboseLevel);
 
+  // Suppress the printing of physics information.
+  if (m_cfg.printPhysicsParameters) {
+    G4HadronicParameters::Instance()->SetVerboseLevel(geantVerboseLevel);
+    G4HadronicProcessStore::Instance()->SetVerbose(geantVerboseLevel);
+    G4EmParameters::Instance()->SetIsPrintedFlag(true);
```

Comment:
looks like this does not compile. are you using a different G4 version?

Reply:
Probably, I mostly use very recent builds from G4... Hopefully there is some kind of `#ifdef` mechanism in G4 for different versions...